docs/topics/signals.rst

```diff
@@ -370,6 +370,27 @@ request_left_downloader
     :param spider: the spider that yielded the request
     :type spider: :class:`~scrapy.spiders.Spider` object
 
+bytes_received
+~~~~~~~~~~~~~~
+
+.. signal:: bytes_received
+.. function:: bytes_received(data, request, spider)
+
+    Sent by the HTTP 1.1 and S3 download handlers when a group of bytes is
+    received for a specific request. This signal might be fired multiple
+    times for the same request, with partial data each time.
```

Comment:
There are 2 ways to return partial data: 

1) return data which is received so far (in this case "partial" means that the response is not fully received yet)
2) return a next received chunk

Would it be possible to clarify in the docs, say that (2) is correct?

Reply:
Updated in e1948b492317eb5b11550d119d91c61b74b3a37f